include/os/linux/zfs/sys/trace_common.h

```diff
@@ -98,7 +96,7 @@
 	"type %u level %u copies %u dedup %u dedup_verify %u nopwrite %u } }"
 
 #define	ZIO_TP_PRINTK_ARGS						\
-	__entry->zio_type, __entry->zio_cmd, __entry->zio_priority,	\
+	__entry->zio_type, __entry->zio_priority,			\
```

Comment:
You should also update ZIO_TP_PRINTK_FMT above.

Reply:
Good spot, done.